storybook/cypress/e2e/image-overlay.cy.ts

```diff
@@ -51,6 +51,9 @@ describe("Image Overlay", () => {
 
   it("should close overlay on click outside", () => {
     openOverlay();
+
+    cy.get("dso-image-overlay.hydrated").shadow().matchImageSnapshot();
```

Comment:
Het maken van het screenshot is verhuisd naar een andere test, wat mij betreft prima maar volgens mij had je voorgesteld om dit weer terug te draaien.

En volgens mij gingen we in deze situatie ook geen `.shadow()` meer doen.